Automattic/jetpack
# Jetpack Social | Fix scheduled post double count

Part of #34182. Right now, when editing an already scheduled post, the scheduled post is double counted.

## Proposed changes:

* Subtract the currently scheduled post from the initial scheduled post count
* Change "Active connections" label to "Enabled connections" to remove confusion

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* For a site with a few social connections added, create a post
* Schedule the post with some connections enabled
* Create another post with some connections enabled
* Now edit the previously scheduled post
* Confirm that the enabled connections for the current post are subtracted from the scheduled count
* Confirm that the label now says "Enabled connections" instead of "Active connections"

<img width="293" alt="Screenshot 2023-11-28 at 5 08 53 PM" src="https://github.com/Automattic/jetpack/assets/18226415/690d7224-6468-4daa-98ad-76751b69bc51">
